buildSrc/src/main/kotlin/Config.kt

```diff
@@ -8,7 +8,7 @@ object Config {
 
     const val compileSdk = 31
     const val targetSdk = 31
-    const val minSdk = 21
+    const val minSdk = 26
```

Comment:
ako neka metoda ili klasa nije podržana na api verziji 21, probaj naći drugo rješenje
Ovo predstavlja najnižu verziju Android OS-a koja može instalirati aplikaciju, sdk 21 je Android 5, možda bi mogli podići na 24 (Android 7), ali 26 (Android 8) je možda previše. 
Provjerim još sa kolegama što oni misle pa javim konačnu odluku, zasad nek stoji ovako.

Reply:
ako se ne varam, minSdk 26 mi treba samo za pozivanje LocalDateTime metoda. Mogu eventualno probati pronaći neku drugu klasu za spremanje datuma i vremena ili ju sam napraviti